joomla/joomla-cms
# [3.10] Fix JS error in pre-update-checker

### Summary of Changes
Removed unneeded description from manifest_cache in json_encoded string as it breaks JS block in <head> if unclean HTML code is set, preventing pre-update-check from running.


### Testing Instructions
Install the [T3-Framework](https://github.com/t3framework/t3/archive/refs/tags/v2.7.6.zip) from JoomlaArt (is a Template-Framework with BS3) and start the pre-update-check.


### Actual result BEFORE applying this Pull Request
Browser Developer-Console tells JS errors and the pre-update-checker works not correctly.


### Expected result AFTER applying this Pull Request
All works as expected.
